docs/quick_start.md

```diff
@@ -2,46 +2,65 @@
 
 We are going to set up a sensor and event-source for webhook. The goal is to trigger an Argo workflow upon an HTTP Post request.
 
+## Install Argo Workflows
 Note: You will need to have [Argo Workflows](https://argoproj.github.io/argo-workflows/) installed to make this work. 
 The Argo Workflow controller will need to be configured to listen for Workflow objects created in `argo-events` namespace. 
   (See [this](https://github.com/argoproj/argo-workflows/blob/master/docs/managed-namespace.md) link.) 
-  The Workflow Controller will need to be installed either in a cluster-scope configuration (i.e. no "--namespaced" argument) so that it has visiblity to all namespaces, or with "--managed-namespace" set to define "argo-events" as a namespace it has visibility to. To deploy Argo Workflows with a cluster-scope configuration you can use this installation yaml file:
+  The Workflow Controller will need to be installed either in a cluster-scope configuration (i.e. no "--namespaced" argument) 
+  so that it has visibility to all namespaces, or with "--managed-namespace" set to define "argo-events" as a namespace it has visibility to.
 
-        kubectl apply -n argo -f https://raw.githubusercontent.com/argoproj/argo-workflows/master/manifests/install.yaml
+  1. Install Argo workflows with a cluster-scope configuration without database and artifact repository
 
+    kubectl apply -n argo -f https://raw.githubusercontent.com/argoproj/argo-workflows/master/manifests/install.yaml
+
+  2. Install Argo workflows with a cluster-scope configuration with database and artifact repository (for local experimentation)
+
+    kubectl apply -n argo -f https://raw.githubusercontent.com/argoproj/argo-workflows/master/manifests/quick-start-postgres.yaml
+
+  You will also need to set up RBAC to allow argo-workflow controller to access argo-events namespace.
```

Comment:
Why is following required? The workflow cluster scope installation should be able to cover all of these already.

Reply:
Because even in cluster scope installation a workflow is not started, it is just created and sits dormant.
I believe that people would like to have a workflow which is executed as it gets created.

Reply:
yes, Cluster scope should pick up all namespace workflows.